### ISSUE TITLE
Something to use or inspire from .  socketcluster.io

### ISSUE THREAD

**Issue by babnik63** [apollostack/apollo#23]
http://socketcluster.io has a good idea and implementation. Please take a look, may help Apollo designers.

**Comment by sebakerckhof** [comment]
@babnik63 I think most MDG developers will be aware of socket-cluster, deepstream, etc. But do they have GraphQL integration (what apollo is all about), can you elaborate on their approach and why it is good?

**Comment by batjko** [comment]
Meatier (a somewhat competing clone of Meteor using a completely different tech stack) is using GraphQL Socketcluster, as well as Webpack, Redux etc.

**Comment by helfer** [comment]
Yes, socketcluster is definitely interesting. At this early stage we're still just looking to HTTP for the transport layer, but when start implementing reactivity, websockets will be part of the picture somehow and using socketcluster seems like a reasonable thing to do in that case.
